components/assetcard/index.css

```
@@ -13,52 +13,84 @@ governing permissions and limitations under the License.
 /* outer container, unstyled */
 .spectrum-AssetCard {
   /* todo: this isn't quite the size from the XD file as 232px is not a size token, so we use 224px */
-  --spectrum-assetcard-asset-size: var(--spectrum-global-dimension-static-size-2800);
+  --spectrum-assetcard-asset-size: 224px;
+  --spectrum-assetcard-background-color: var(--spectrum-gray-200);
+  --spectrum-assetcard-asset-animation-duration: var(--spectrum-animation-duration-100);
+  --spectrum-assetcard-assetcontainer-border-size: 1px;
```

```suggestion
  --spectrum-assetcard-asset-container-border-size: 1px;
```

Since this is a new property, should we use the dash in the name to keep it human-readable?